### PR TITLE
Add md5 checksum to videoDecodeRGB

### PR DESCRIPTION
**Pull request by paveltc**
Added the MD5 checksum and MD5 checksum check to videoDecodeRGB

### CODE REVIEW

File: samples/videoDecodeRGB/videodecrgb.cpp
Comment (AryanSalmanpour) [inline samples/videoDecodeRGB/videodecrgb.cpp:244]: change the condition to (`b_generate_md5 && convert_to_rgb)`

File: samples/videoDecodeRGB/videodecrgb.cpp
Comment (AryanSalmanpour) [inline samples/videoDecodeRGB/videodecrgb.cpp:67]: remove the extra line

File: samples/videoDecodeRGB/videodecrgb.cpp
Comment (AryanSalmanpour) [inline samples/videoDecodeRGB/videodecrgb.cpp:500]: add `<< std::endl` before `;` at the end of this line and line `502`

**Review by AryanSalmanpour — CHANGES_REQUESTED** [review]

File: utils/rocvideodecode/roc_video_dec.cpp
Comment (rrawther) [inline utils/rocvideodecode/roc_video_dec.cpp:1002]: remove empty line here and 1004

**Review by rrawther — COMMENTED** [review]
Added minor comment. Can be merged after Aryan approves it

**Pull request closed without merging**
